Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:10
```

Comment:
Any particular reason you're not using Node v12?

Reply:
I'm running node 10 still.